test/k8s-integration/driver.go

```diff
@@ -65,11 +77,18 @@ func installDriver(goPath, pkgDir, stagingImage, stagingVersion, deployOverlayNa
 	if err != nil {
 		return fmt.Errorf("failed to deploy driver: %v", err)
 	}
-
+	klog.Infof("Deploying driver")
+	fmt.Printf("Deploying driver")
 	// TODO (#139): wait for driver to be running
-	time.Sleep(time.Minute)
-	statusCmd := exec.Command("kubectl", "describe", "pods", "-n", driverNamespace)
-	err = runCommand("Checking driver pods", statusCmd)
+	if platform == "windows" {
+		klog.Infof("Test waiting for 15 mins")
+		time.Sleep(15 * time.Minute)
+	} else {
+		time.Sleep(1 * time.Minute)
+	}
+	out, err = exec.Command("kubectl", "describe", "pods", "-n", driverNamespace).CombinedOutput()
+	klog.V(2).Infof("describe pods \n %s", string(out))
+	fmt.Printf("describe pods \n %s", string(out))
```

Comment:
Same thing both klog and fmt?

Reply:
from build-log.txt, I cannot see this log with klog only. It works with fmt.Printf. Is it normal?